pages/date-picker/positioning.page.tsx

```diff
@@ -14,7 +14,7 @@ export default function DatePickerScenario() {
         style={{
           inlineSize: '180px',
           marginInlineStart: '300px',
-          marginBlockStart: '300px',
+          marginBlockStart: '285px',
```

Comment:
The page content overflows the window horizontally, which leads to a horizontal scrollbar appearing at the bottom of the page, taking up 15px of vertical space and altering the test results.

Verified that this page is not used for visual regression.